target/arc/mmu-v6.c

```diff
@@ -606,7 +584,7 @@ page_table_traverse(CPUARCState *env,
                 qemu_log_mask(CPU_LOG_MMU, "[MMUV3] PTE seems invalid\n");
             }
 
-            mmu_fault_status = (l & 0x7);
+            env->mmu.v6.fault_status = (l & 0x7);
```

Comment:
Can you use a different name here but not 'l' as it is difficult to see it/trace it.